notebook_review_process.md

```diff
@@ -18,17 +18,18 @@ For authors: consider these checklists requirements for your code.
       - if not, is that justified
 - Does it include work linked to a buzzword:
 	- big data, spectroscopy, time domain, forced photometry, cloud
-- Has each NASA archive been given option to comment on modules for their relevant data access?\
+- Has each NASA archive been given the option to comment on modules for their relevant data access?\
   	- TODO: a preferred contact method for each archive should be listed here, ie., archive helpdesk, NN slack channel #fornaxdev-daskhub? ??? 	
 	- Is archival data accessed in the most efficient way according to that archive?
 ## Tech Review Checklist
 - Documentation:
 	- Is every function documented?
 	- Does it follow the style guide? https://github.com/spacetelescope/style-guides/blob/master/guides/jupyter-notebooks.md
    	- Do all code cells have corresponding narratives/comments?
-   	- Include information about runtime on fiducial Fornax server
-   	- Include information about which "image" the notebook uses when loggin into Fornax, ie., "Astrophysics default image"
+   	- Include information about runtime on the fiducial Fornax server
+   	- Include information about which "image" the notebook uses when logging into Fornax, ie., "Astrophysics default image"
```

Comment:
```suggestion
   	- Include information about which server type and environment to choose when logging in to Fornax and the notebook's expected runtime given that setup. For example: 
   	    - "As of 2024 August, this notebook takes about 3 minutes to run to completion on Fornax using Server Type: 'Standard - 8GB RAM/4 CPU' and Environment: 'Default Astrophysics' (image)."
```

Some things on Fornax have changed, so I'm trying to help this be more clear... not sure how well I've done, but here's my thinking:

- The available server types have changed and could change again, so it would be good for the notebooks to be more specific about what was used or what would be a good choice. ("Fiducial" might mean something different in 6 months.) 
- The name of the default astro image has changed, and the label next to that dropdown is "Environment" (with no mention of "image" anywhere). That's confusing because "environment" can also refer to a kernel or conda env. Hopefully a statement like the example I added here (taken from the spectroscopy notebook) will be more clear for the end user.